packages/react-component-library/package.json

```diff
@@ -70,7 +70,7 @@
     "@types/react-dom": "^16.9.5",
     "@types/react-select": "^3.0.10",
     "@types/react-transition-group": "^4.2.3",
-    "@types/storybook__react": "^5.2.1",
+    "@types/storybook__react": "^4.0.2",
```

Comment:
This feels strange - I thought I bumped this forward?

Reply:
https://github.com/Royal-Navy/standards-toolkit/commit/ee5d2342b498bea3db931ecb58c83695b7caac96